script/Arcueid/Day 3/03_00_ARC03_5_1.txt

```diff
@@ -345,17 +359,18 @@ Tohno Shiki should have known that humans could be torn apart just as easily.
 [sha:0a0493252b546d2720ed8cf4ad169d14085c5e97]{
 -- Page 266, Offset 4726.
 -- 俺は、そんな事を考えもしないで、ただ無責任に暮らしてしまった。
-I'd been living irresponsibly, not even stopping to consider that possibility.
+I have been living irresponsibly, never even stopping to consider that possibility.
 }
 [sha:3ddd3378bdbc6937d432e2dd754213f44e8d5b99]{
 -- Page 267, Offset 4727.
 -- ……自分が、簡単に何かを殺してしまう危険な人間なら。
-I can kill things so easily... I'm a dangerous human being.
+// Swapped with below.
+I should have blinded myself, or endeavored to live a life where I'd never see another soul.
 }
 [sha:9815145bb18c84e837728fc603d7d149f503f3c9]{
 -- Page 267, Offset 4728.
 -- 俺はこの目を潰すか、誰とも会わないような生活をするべきだったのに。
-I should have blinded myself, or endeavored to live a life where I'd never see another soul.
+Because... I am a dangerous person who can easily kill anything.
```

Comment:
This line still feels kinda dumb but not sure how much can be done about it.
> After all, I'm a danger to others... I could accidentally kill with just a touch. 

Or revert the order and lead off with a reordered first sentence
> I'm a dangerous individual. It takes so little for me to accidentally end a life.

> It takes so little to end a life... I'm too dangerous to be 
>